static/app/components/profiling/flamegraph.tsx

```diff
@@ -91,9 +91,8 @@ function Flamegraph(props: FlamegraphProps): ReactElement {
         />
       </FlamegraphZoomViewMinimapContainer>
       <FlamegraphZoomViewContainer>
-        <ProfileDragDropImport onImport={onImport}>
+        <ProfileDragDropImport onImport={props.onImport}>
           <FlamegraphZoomView
-            key={`${profiles.traceID}-${flamegraph.profileIndex}`}
```

Comment:
Why was this removed?

Reply:
It was more of a "hack" before and it's annoying because it makes the underlying hooks inside the view behave slightly differently (like useEffect fires on mount each time they key changes instead of just prop changes etc), so it was adding to the confusion

Reply:
Ah, let me check that.

Reply:
Manually reset this now when flamegraph renderer changes, but I will centralize all of this logic in a separate PR, these side effects and all are becoming painful to manage and track